README.rst

```diff
@@ -401,3 +401,28 @@ Database example: ::
     messages=no
     randomWait=360
     status=enabled
+
+Notifications
+=============
+
+Mail for root user can be forwarded to external addresses.
+
+Properties:
+
+- `EmailAddress`: comma-separated list of mail addresses; messages sent to root user will be forwarded to listed addresses
+- `KeepMessageCopy`: can be `yes` or `no`; if set to `yes`, messages will be always delivered also to local root mail folder
+- `SenderAddress`: a valid mail address; if not empty, messages sent by root (like cron notifications) will be sent using
+  the specified address. A good value could be: `no-reply@<domain>` (where `<domain>` is the domain of the server).
+  If not set, messages will be sent using `root@<fqdn>` as sender address.
+
+Database example: ::
+
+ root=configuration
+    EmailAddress=myuser@nethserver.org
+    KeepMessageCopy=yes
+    SenderAddress=no-reply@nethserver.org
+
+Usage: ::
+
+ config setprop root Emailaddress myuser@nethserver.org SenderAddress no-reply@nethserver.org
+ signal-event nethserver-mail-smarthost-save
```

Comment:
If we provide the "abstract" API in nethserver-base  we must define an event too, without relying on another package

Reply:
I agree, you could call `profile-modify`event, but it doesn't expand postfix templates.
I can add a bunch of postifx templates and actions from smart-host package inside it, but then, we are again at the starting point :)

Reply:
And since nethserver-base is the main package, it shouldn't rely on templates or actions from dependent packages.